taker-frontend/src/components/AlertBox.tsx

```diff
@@ -1,18 +1,34 @@
-import { Alert, AlertDescription, AlertIcon, AlertStatus, AlertTitle } from "@chakra-ui/react";
+import { ExternalLinkIcon } from "@chakra-ui/icons";
+import { Alert, AlertDescription, AlertIcon, AlertStatus, AlertTitle, Link } from "@chakra-ui/react";
 import * as React from "react";
+import { Link as ReachLink } from "react-router-dom";
 
 interface AlertBoxProps {
     title: string;
     description: string;
     status?: AlertStatus;
+    reachLinkTo?: string;
 }
 
-export default function AlertBox({ title, description, status = "error" }: AlertBoxProps) {
+export default function AlertBox({ title, description, status = "error", reachLinkTo }: AlertBoxProps) {
     return (
         <Alert status={status}>
             <AlertIcon />
             <AlertTitle mr={2}>{title}</AlertTitle>
-            <AlertDescription>{description}</AlertDescription>
+            {reachLinkTo
```

Comment:
An alternative option would be to use `{children}` here and let the user of the component decide on what additional elements should be added to the alert box description. 
E.g. like in here: https://github.com/itchysats/itchysats/blob/f3617e51e088265bc4fd891bb9d2a5ff3dbae1b5/taker-frontend/src/components/FundingRateTooltip.tsx#L5-L25

Reply:
yeah, I totally thought about this, but was not sure if I can achieve exactly what I wanted so I did not invest time for now.
Happy to see it in a follow-up.